### PR TITLE
Edit Template Files - TFPluginDocs

### PR DESCRIPTION
**Pull request by sheneska**
### Description

This PR adds `data fields`, where appropriate to the template files for runtime class and namespace resources. 

### Acceptance tests
- [ ] Have you added an acceptance test for the functionality being added?
- [ ] Have you run the acceptance tests on this branch?

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### Release Note
Release note for [CHANGELOG](https://github.com/hashicorp/terraform-provider-kubernetes/blob/main/CHANGELOG.md):

```release-note
...
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

File: docs/data-sources/all_namespaces.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ In Kubernetes, namespaces provide a scope for names and are intended as a way to
 ## Example Usage
 
 ```terraform
+# Copyright (c) HashiCorp, Inc.
```
Comment (jrhouston) [inline docs/data-sources/all_namespaces.md:17]: Not sure how all these copyright comments got added but they shouldn't be in the example snippets?

Comment (sheneska) [reply docs/data-sources/all_namespaces.md:17]: Right, Not sure if that's intended behavior - I'll raise it with the devex team but basically when the files are migrated to the new structure, it picks up everything from the `.tf` file including the header.

**Review by jastr945 — APPROVED** [review]
I went through the changes, although i don't have much context, the updates made sense to me.

File: docs/resources/api_service.md [hunk 2]
```diff
@@ -9,6 +9,67 @@ description: |-
 
 An API Service is an abstraction which defines for locating and communicating with servers.
 
+<!-- schema generated by tfplugindocs -->
+## Schema
```
Comment (jrhouston) [inline docs/resources/api_service.md:13]: This autogenerated schema section is now duplicating the information in the [argument reference below](https://github.com/hashicorp/terraform-provider-kubernetes/blob/706e53b1bd95d58c9a4aa1e3cca2fe7495843e3d/docs/resources/priority_class_v1.md#argument-reference) – so I think you'll need to remove all those sections.

**Review by arybolovlev — APPROVED** [review]

**Pull request merged**